File: CS/Form1.cs
using System;
using System.Drawing;
using System.Windows.Forms;
using DevExpress.XtraCharts;

namespace ChartEmptyPie {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void simpleButton1_Click(object sender, EventArgs e) {
            // Remove all Series from chart 
            chartControl1.Series.Clear();

            // Simulate "No data" state
            Series series = new Series("Empty Series", ViewType.Pie);

            series.Label.PointOptions.PointView = PointView.Argument;
            ((PieSeriesLabel)series.Label).Position = PieSeriesLabelPosition.Inside;
            ((PieSeriesLabel)series.Label).Font = new Font("Arial", 40);
            series.Points.Add(new SeriesPoint("No data", new object[] { 0.2 }));

            chartControl1.Series.Add(series);
            ((SimpleDiagram)chartControl1.Diagram).EqualPieSize = true;
            chartControl1.Legend.Visible = false;
        }
    }
}

File: VB/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraCharts

Namespace ChartEmptyPie
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
			' Remove all Series from chart 
			chartControl1.Series.Clear()

			' Simulate "No data" state
			Dim series As New Series("Empty Series", ViewType.Pie)

			series.PointOptions.PointView = PointView.Argument
			CType(series.View, PieSeriesView).SizeAsPercentage = 80
			CType(series.Label, PieSeriesLabel).Position = PieSeriesLabelPosition.Inside
			CType(series.Label, PieSeriesLabel).Font = New Font("Arial", 40)
			series.Points.Add(New SeriesPoint("No data", New Object() {0.2}))

			chartControl1.Series.Add(series)
			chartControl1.Legend.Visible = False
		End Sub
	End Class
End Namespace